Runner.php

```diff
@@ -63,6 +65,20 @@ public function queuePaths($dir, $prefix = '', $exclusions = []) {
         }
     }
 
+    public function prefixCodeDirectory($configRulesets)
+    {
+        $officialPhpRulesets = explode(',', Runner::RULESETS);
+        $configRulesets = explode(',', $configRulesets);
+
+        foreach ($configRulesets as &$r) {
+            if (!in_array($r, $officialPhpRulesets)) {
+                $r  = "/code/$r";
```

Comment:
I think it might be worth it to detect the case where someone enters an absolute path (ie. don't prepend `/code` if the string starts with `/`)